### ISSUE TITLE
Show Snapshots

### ISSUE THREAD

**Issue by maikito26** [dbagchee/helix-preset-viewer#35]
This one is a thinker.  I would prefer tabs at the top with the snapshots able to be selected.
- I would like the tab to show an indicator if the snapshot has something configured different than snapshot 0
- I would like the blocks to then indicate that it has parameter differences

Another thought I had was maybe when we click on a block, it would draw the parameters.  And each snapshot could be its own column in the resultant table.

**Comment by dbagchee** [comment]
Yes will need to think on this one - we probably need to add an on/off state for the blocks too so you can see which blocks are currently on in a preset - this would reflect in snapshots as well
